### DISCUSSION TITLE
Content update is not working

### DISCUSSION THREAD

**Discussion by pranavirabatti** [Sunbird-Lern/Community#9] [Issues]
Request URL: host/action/content/v3/hierarchy/update

Request Payload:
https://pastebin.com/MpqDU7iy

```
{
  "id": "api.hierarchy.update",
  "ver": "1.0",
  "ts": "2022-06-02T13:31:41.221Z",
  "params": {
    "resmsgid": "56399550-e278-11ec-8e8d-d5b6ee60993a",
    "msgid": null,
    "status": "failed",
    "err": "CLIENT_ERROR",
    "errmsg": "Validation Errors."
  },
  "responseCode": "CLIENT_ERROR",
  "result": {
    "messages": [
      "subject range data is empty from the given framework."
    ]
  }
}
```

@amit-tarento, @keshavprasadms, @reshmi-nair, @HarishGangula what is missing here? We are passing all the mandatory fields and even then I'm getting this. Anything we left to add when creating the framework?

cc: @coolbung, @Renuka-S @komalm

**Comment by amitec006** [comment]
@pranavirabatti This is not a Sunbird Lern BB related issue. Please raise this in  Sunbird [Knowlg](https://github.com/Sunbird-Knowlg/Community/discussions) discussion forums.

**Reply by rhwarrier** [reply]
@pranavirabatti  please take this up with Knowlg as indicated above. Locking this thread for now.
